remoting/etcdv3/listener.go

```diff
@@ -0,0 +1,216 @@
+package etcdv3
+
+import (
+	"sync"
+	"time"
+)
+
+import (
+	"github.com/apache/dubbo-go/common/logger"
+	"github.com/apache/dubbo-go/remoting"
+)
+
+import (
```

Comment:
this import block should be in second.

Reply:
Thx, i had reformat import block in latest commit.